app/views/devise/registrations/new.html.haml

```diff
@@ -26,15 +26,17 @@
     .col-md-8= f.password_field :password_confirmation, :class => 'form-control'
 
   .form-group
-    .col-md-offset-2.col-md-8
-      = f.check_box :tos_agreement
-      I agree to the
-      = succeed "." do
-        = link_to 'Terms of Service', url_for(:action => 'tos', :controller => '/policy')
-  .form-group
-    .col-md-offset-2.col-md-8
-      = f.check_box :newsletter, :checked => true
-      Subscribe to the #{ENV['GROWSTUFF_SITE_NAME']} newsletter
+    .col-md-offset-2.col-md-8.checkbox
+      %label
+        = f.check_box :tos_agreement
+        I agree to the
+        = succeed "." do
+          = link_to 'Terms of Service', url_for(:action => 'tos', :controller => '/policy')
```

Comment:
I just checked this locally - clicking "I agree to the" checks the checkbox, and clicking "Terms of Service" follows the link. Firefox 38 and Chromium 43 on Linux. Is that enough, or do we need further checks?